packages/javascript/bh-shared-ui/src/components/ExploreTable/useExploreTableRowsAndColumns.tsx

```diff
@@ -75,23 +75,17 @@ const useExploreTableRowsAndColumns = ({
 
     const handleSort = useCallback(
         (sortByColumn: keyof MungedTableRowWithId) => {
-            if (sortByColumn) {
-                if (sortBy === sortByColumn) {
-                    switch (sortOrder) {
-                        case 'desc':
-                            setSortOrder('asc');
-                            break;
-                        case 'asc':
-                            setSortOrder('desc');
-                            break;
-                        default:
-                            setSortOrder('desc');
-                            break;
-                    }
-                } else {
-                    setSortBy(sortByColumn);
-                    setSortOrder('asc');
-                }
+            if (!sortByColumn || sortByColumn !== sortBy) {
+                // first sort of a new column
+                setSortBy(sortByColumn);
```

Comment:
TY for adding the comments!! Makes it super easy to follow and im sure future us will appreciate it even more hahah

Im curious about the `!sortByColumn ||` check in this if statement, if `sortByColumn` is undefined (which I highly doubt it ever would be), would we be effectively resetting the `sortBy` column and setting `sortOrder` to 'asc'?